packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemWrapperStyles.ts

```diff
@@ -119,6 +118,20 @@ export const menuItemWrapperStyles: ComponentSlotStylesPrepared<MenuItemWrapperS
         }),
       }),
 
+      ...(active && {
+        '[data-tabs="true"]': {
```

Comment:
This is not the solution that I like, but it solves the issue. Using React context to pass down a value will be more complex and more costly in terms of perf.

Fela also doesn't support `:global()` that could be a workaround for this case.